docs/fundamentals/code-analysis/style-rules/language-rules.md

```diff
@@ -0,0 +1,1920 @@
+---
+title: Code style language rules
+ms.date: 09/25/2020
+ms.topic: reference
+author: gewarren
+ms.author: gewarren
+dev_langs:
+- CSharp
+- VB
+helpviewer_keywords:
+- language code style rules [EditorConfig]
+- language rules
+- EditorConfig language conventions
+---
+# Language rules
+
+Code style language rules affect how various constructs of .NET programming languages are used, for example, modifiers and parentheses. They fall into the below categories:
+
+- [.NET style rules](#net-style-rules): Rules that apply to both C# and Visual Basic. The EditorConfig option names for these rules start with `dotnet_style_` prefix.
+- [C# style rules](#c-style-rules): Rules that are specific to C# language only. The EditorConfig option names for these rules start with `csharp_style_` prefix.
+
+> [!TIP]
```

Comment:
I would just delete this entire tip. We don't really use this pattern in the .NET docs.

Reply:
Sounds good. I will also delete the `code-language-picker.png` that I ported as part of porting this tip.